packages/block-library/src/video/edit.js

```diff
@@ -219,7 +219,7 @@ function VideoEdit( {
 						attributes={ attributes }
 					/>
 					<MediaUploadCheck>
-						<BaseControl className="editor-video-poster-control">
+						<div className="editor-video-poster-control">
```

Comment:
## Testing instructions

In the block editor, add a Video block and see the block inspector.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/b68b845b-98e0-438d-aff7-2d12a04b58f3" alt="Video block inspector, before" width="274">|<img src="https://github.com/user-attachments/assets/14246f70-de09-4f5e-8438-b0cc487fd0a5" alt="Video block inspector, after" width="274">|

Switched the unnecessary BaseControl to a `div`.